api/http/handler/websocket/pod.go

```diff
@@ -150,9 +150,10 @@ func (handler *Handler) hijackPodExecStartOperation(
 	err = <-errorChan
 	if websocket.IsUnexpectedCloseError(err, websocket.CloseGoingAway, websocket.CloseNoStatusReceived) {
 		log.Printf("websocket error: %s \n", err.Error())
+		return nil
 	}
 
-	return nil
+	return &httperror.HandlerError{http.StatusInternalServerError, "Unable to start exec process inside container", err}
```

Comment:
Whose going to see this error?  if it's bubbled up to an end user, does it provide accurate and meaningful information that will help them to resolve the problem?

Reply:
its the same error as what existed before.
Not changing it as its out of context of ticket.

I believe this will be seen if `StartExecProcess` throws an error first (before the other go routines above it).